policies/lib/string_utils.rego

```diff
@@ -21,9 +23,14 @@ quoted_values_string(value_list) = result {
 	result := concat(", ", quoted_list)
 }
 
-result_helper(rule_metadata, failure_sprintf_params) = result {
+result_helper(chain, failure_sprintf_params) = result {
```

Comment:
We might need a test for this

Reply:
Ack. I think we should also move it out of `string_utils.rego` because it doesn't really feel like a string utility.